Project-Sloth/ps-housing
# Person who got access to a property got access to Garage now

# Overview
*Person who got access to a property got access to Garage know*

# Testing Steps
*let someone else buy a house, let him give u access to the house, then try accessing the existing garage.*

- [YES] Did you test the changes you made?
- [YES] Did you test core functionality of the script to ensure your changes do not regress other areas?
- [YES] Did you test your changes in multiplayer to ensure it works correctly on all clients?
